src/main/resources/assets/tconstruct/lang/zh_cn.json

```diff
@@ -2638,6 +2809,10 @@
   "modifier.tconstruct.the_one_probe": "The One Probe",
   "modifier.tconstruct.the_one_probe.flavor": "科学化！",
   "modifier.tconstruct.the_one_probe.description": "当手持或者附加于头盔时，能够显示指向方块的信息",
+  "modifier.tconstruct.headlight": "Headlight",
```

Comment:
```suggestion
  "modifier.tconstruct.headlight": "头灯",
```

Reply:
this is a Compat Modifiers, compat with Headlight mod that doesn't have a Chinese localization name yet.